### PR TITLE
nfs-ganesha: change CMAKE_BUILD_TYPE, add checks

### PR DESCRIPTION
**Pull request by alimaredia**
- change CMAKE_BUILD_TYPE to RelWithDebInfo
- include checks to make sure packages are installed
on the machines and necessary FSAL packages are built
- update $CEPH_VERSION to latest luminous

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

File: nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@
       - string:
           name: CEPH_VERSION
           description: "The version of Ceph to specify for installing ceph libraries"
-          default: "12.2.2"
+          default: "12.2.3"
```
Comment (ajarr) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: The latest stable isn't built with 12.2.4? I remember you reporting so.

**Comment by alimaredia** [comment]
@ajarr I had mispoken. The latest packages on download.ceph.com were not built with 12.2.4, they were built with 12.2.3. I thought 12.2.3 was the latest version, and didn't realize how quickly 12.2.4 came out which was just 2 weeks after 12.2.3. 

I asked Jeff and Matt if anything went into 12.2.4 that would impact the way nfs-ganesha uses ceph and they said no. If there is something they missed let me know and I will rebuild the V2.5-stable debs/rpms with 12.2.4. 

After talking with Matt, there is an important rgw related fix that should be in 12.2.5. When 12.2.5 is out I will make sure that the 2.5 packages are rebuilt with it. Also I will build 2.6 packages as soon as this PR is merged.

**Comment by ajarr** [comment]
LGTM

**Comment by ajarr** [comment]
retest this please

**Pull request merged**